lambdas/tests/unit/services/test_bulk_upload_service.py

```diff
@@ -200,7 +201,10 @@ def test_handle_sqs_message_happy_path(
     mock_pds_validation,
     mock_ods_validation,
 ):
+    # given
```

Comment:
Apologies, I know I added this in as an example when we paired, but for consistency can we remove the given, when, then comments please?